falco.yaml

```diff
@@ -613,7 +613,7 @@ syscall_event_drops:
   max_burst: 1
   simulate_drops: false
 
-# [Experimental] `metrics`
+# [Stable] `metrics`
```

Comment:
@FedeDP @leogr WDYT? Other than that one odd behavior where the timer did not work when using http output (which Federico fixed) I have not heard of more issues when using the metrics feature, have you? It is working well for me in production.

Proposing to bump to Stable, which does not mean that there may be no additional bugs, it rather signals we are supporting `metrics` now as stable feature going forward.

Reply:
I think this is good 2 go! Agree with you.
@leogr more thoughts?